tests/fault_tolerance/utils/metrics.py

```
@@ -0,0 +1,119 @@
+from multiprocessing import Process
+
+import psutil
+import pytest
+
+from dynamo.runtime import dynamo_worker
+from tests.fault_tolerance.utils.circus_controller import CircusController
+from tests.utils.managed_process import ManagedProcess
+
+
+def run_metrics_process(log_dir):
+    asyncio.run(get_metrics(log_dir))
```

Pylint (3.3.7)

[error] No value for argument 'log_dir' in function call

(E1120)